src/components/ContextChart/index.js

```diff
@@ -135,10 +139,11 @@ class ContextChart extends Component {
         >
           {annotations}
           <LineCollection
-            series={contextSeries}
+            series={series.map(s => ({ ...s, drawPoints: false }))}
```

Comment:
Seems that drawPoints will override all the user specified behaviour if any provided, is it by design for `ContextChart` not being able to render points?

Reply:
Correct -- ContextChart does not render points.